docs/en/connector-v2/source/MySQL-CDC.md

```diff
@@ -129,7 +129,7 @@ When an initial consistent snapshot is made for large databases, your establishe
 - `interactive_timeout`: The number of seconds the server waits for activity on an interactive connection before closing it. See [MySQL’s documentation](https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_interactive_timeout) for more details.
 - `wait_timeout`: The number of seconds the server waits for activity on a non-interactive connection before closing it. See [MySQL’s documentation](https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_wait_timeout) for more details.
 
-*For more database settings see [Debezium MySQL Connector](https://debezium.io/documentation/reference/1.6/connectors/mysql.html#setting-up-mysql)*
+*For more database settings see [Debezium MySQL Connector](https://debezium.io/documentation/reference/2.6/connectors/mysql.html#setting-up-mysql)*
```

Comment:
CDC dependent debezium `1.6.4.Final`. Changing the documentation link to version `2.6` seems to cause confusion.
You may try imitating #6016.